nvflare/fuel/f3/streaming/byte_streamer.py

```diff
@@ -172,6 +169,9 @@ def _transmit(self, task: TxTask, final=False):
         task.stream_future.set_progress(task.offset)
 
     def _stop_task(self, task: TxTask, error: StreamError = None, notify=True):
+        with self.map_lock:
+            self.tx_task_map.pop(task.sid, None)
```

Comment:
@chesterxgchen these parts for memory leak.

Previous code did NOT pop these so the reference of task was kept.
Thus Task data was kept, thus memory not getting garbage collected.